apps/bbsync/tests/test_cc.py

```diff
@@ -179,12 +179,14 @@ def test_notaffected(self, cc, embargoed):
         """
         flaw = FlawFactory(embargoed=embargoed)
         affect1 = AffectFactory(
-            flaw=flaw, affectedness=Affect.AffectAffectedness.NOTAFFECTED
+            flaw=flaw,
+            affectedness=Affect.AffectAffectedness.NOTAFFECTED,
         )
         affect2 = AffectFactory(
             flaw=flaw,
             affectedness=Affect.AffectAffectedness.AFFECTED,
-            resolution=Affect.AffectResolution.WONTFIX,
```

Comment:
This test is being modified in two commits for no apparent reason

Reply:
These have been squashed.